payment_success.php

```diff
@@ -4,14 +4,12 @@
 if(!isset($_SESSION["uid"])){
 	header("location:index.php");
 }
-$trx_id = $_GET["tx"];
-$p_st = $_GET["st"];
-$amt = $_GET["amt"];
-$cc = $_GET["cc"];
-$cm = $_GET["cm"];
-$c_amt = $_COOKIE["ta"];
-
-
+// $trx_id = $_GET["tx"];
+// $p_st = $_GET["st"];
+// $amt = $_GET["amt"];
+// $cc = $_GET["cc"];
+//  $cm = $_GET["cm"];
+//  $c_amt = $_COOKIE["ta"];
```

Comment:
Was this really necessary? I think there is a problem with your machine, because none of us is experiencing such errors except you. And those scripts have a task to perform in the future, so commenting them out is not a solution.

Reply:
removed already but I have saved them somewhere else in case needed later.